atlp-rwanda/atlp-devpulse-bn
# #149 Create email templates

### PR Description
This PR is for creating the email templates to be used while sending emails notifying users different action

### Description of tasks that were expected to be completed
-  Design and implement reusable email templates for notifications and action-based emails, such as account updates, password resets, and verification emails. These templates will ensure consistency and improve the user experience for all email communications.

### How has this been tested?
- Clone repo
- Checkout branch `revamp-email-templates`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Then you will create account  and check your email inbox to see the email verification.



### Track PR (issue number & link)
[149 - Frontend](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/149)

### Screenshots
  
![Screenshot 2024-10-02 173630](https://github.com/user-attachments/assets/559215d4-1412-4fa1-b229-1ebc1e2a0d10)
![Screenshot 2024-10-02 173642](https://github.com/user-attachments/assets/a300aa39-c9d1-4045-adb2-f1e01c34edd1)
